src/com/directi/training/codesmells/refactored/chess/ChessBoard.java

```diff
@@ -96,17 +98,16 @@ public boolean isEmpty(Position position)
 
     private Cell getCell(Position position)
     {
-        return _board[position.getRow()][position.getColumn()];
+        return _board[position.row][position.column];
     }
 
     //Dead-Code Code Smell fixed by removing getPlayerName and printMove methods (and also toString of Position),
     // as well as player 1 and player 2 fields.
 
+    //Fixed Duplicate Code across classes code smell by calling isEmpty method
```

Comment:
across classes sound wrong. Either we can omit it or change it to across class/ across methods of a class. Same change is required in readme.